### DISCUSSION TITLE
cant join any server in the launcher

### DISCUSSION THREAD

**Discussion by bjorne69** [space-wizards/space-station-14#24053] [Technical]
So i tried to join a ss 14 server but it aint working, i cant join ANY server. i have tried reinstalling, enabling compatability mode, downloading the launcher from the website, even trying to create a alt account.

**Comment by PJB3005** [comment]
Please post what errors you're getting.

**Reply by bjorne69** [reply]
i cant join any server, when i just try to enter a server a small loading screen appears then i just get booted to the launcher again.

**Reply by bjorne69** [reply]
https://github.com/space-wizards/space-station-14/assets/156553679/602b657b-342d-4635-ba95-4c714d82a1e2

**Reply by PJB3005** [reply]
Please enable logging in the launcher and send us the log files

**Reply by bjorne69** [reply]
[launcher.log](https://github.com/space-wizards/space-station-14/files/13932058/launcher.log)
[client.stdout.log](https://github.com/space-wizards/space-station-14/files/13932075/client.stdout.log)
[client.stderr.log](https://github.com/space-wizards/space-station-14/files/13932076/client.stderr.log)
I think these are the right files.

**Reply by PJB3005** [reply]
Is your `%appdata%/space station 14/data/keybinds.yml` messed up maybe? Try renaming it.

**Reply by bjorne69** [reply]
Thank you man, i realised i've done some wrong keybinds but now it finally works.
